server/server.js

```diff
@@ -0,0 +1,21 @@
+import dotenv from "dotenv";
+import express from "express";
+import { fileURLToPath } from "url";
+import path, { dirname } from "path";
```

Comment:
if you deconstruct "path", you don't neet to import "path", just add "join" to the object